### DISCUSSION TITLE
Whither ignore_failure?

### DISCUSSION THREAD

**Discussion by chakathepakuni** [excid3/noticed#400] [Q&A] [answered]
Curious for insight into the removal of the ignore_failure option in Noticed 2.0. It was useful in development as Twilio sometimes barfs on test phone numbers. Before I start forking and hacking, is there anything I missing?

**Comment by excid3** [comment, marked as answer]
Since they're just jobs, my intention was to revisit that and use `discard_on` and `rescue_from` in the delivery methods but I forgot about it.
